deployment/multiarch-agent-deployment/ServiceGroupRoot/RolloutSpecs/RolloutSpecs.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "https://ev2schema.azure.net/schemas/2020-01-01/rolloutSpecification.json",
+    "ContentVersion": "1.0.0.0",
+    "RolloutMetadata": {
+      "ServiceModelPath": "ServiceModels//Public.ServiceModel.json",   
+      "ScopeBindingsPath": "ScopeBindings//Public.ScopeBindings.json", 
+      "Name": "ContainerInsightsAgent",
+      "RolloutType": "Major",
+      "BuildSource": {
+        "Parameters": {
+          "VersionFile": "buildver.txt"
+        }
+      },
+      "Notification": {
+        "Email": {
+          "To": "amagraw@microsoft.com"
```

Comment:
leaving comment as a reminder to switch this back to omscontainers before merging